odata_webservice/src/main/java/com/sdl/odata/webservice/TomcatConfiguration.java

```diff
@@ -70,11 +70,11 @@ private Connector createSslConnector() {
             connector.setSecure(true);
             connector.setPort(Integer.parseInt(httpsPort));
             protocol.setSSLEnabled(true);
-            protocol.setKeystoreFile(keystore.getAbsolutePath());
-            protocol.setKeystorePass(keystorePasswd);
-            protocol.setTruststoreFile(keystore.getAbsolutePath());
-            protocol.setTruststorePass(truststorePasswd);
-            protocol.setKeyAlias(keyAlias);
+//            protocol.setKeystoreFile(keystore.getAbsolutePath());
```

Comment:
Can this be removed?

Reply:
it's captured on confluence in a TODO list to fix later